apps/src/applab/ImportScreensDialog.jsx

```diff
@@ -269,7 +269,7 @@ export const ImportScreensDialog = React.createClass({
 
 export default connect(
   state => ({
-    isOpen: state.screens.isImportingScreen && state.screens.importProject.fetchedProject,
+    isOpen: !!(state.screens.isImportingScreen && state.screens.importProject.fetchedProject),
```

Comment:
This just fixes a React warning where we were expecting a bool and getting an obj.